File: OLSource1/v1/VS_officedev/codesnippet/CSharp/how-to--programmatically-sort-data-in-worksheets_4.cs
            Excel.ListObject fruitList = 
                 ((Excel.Worksheet)Application.ActiveSheet).
                     ListObjects.Add(Microsoft.Office.Interop.Excel.XlListObjectSourceType.xlSrcRange,
                     Application.get_Range("A1", "B3"), 
                     missing, Microsoft.Office.Interop.Excel.XlYesNoGuess.xlNo); 
                fruitList.Range.Sort(
                    fruitList.ListColumns[1].Range, Excel.XlSortOrder.xlAscending,
                    fruitList.ListColumns[2].Range, missing, Excel.XlSortOrder.xlAscending,
                    missing, Excel.XlSortOrder.xlAscending,
                    Excel.XlYesNoGuess.xlYes, missing, missing,
                    Excel.XlSortOrientation.xlSortColumns,
                    Excel.XlSortMethod.xlPinYin,
                    Excel.XlSortDataOption.xlSortNormal,
                    Excel.XlSortDataOption.xlSortNormal,
                    Excel.XlSortDataOption.xlSortNormal);

File: OLSource1/v1/VS_officedev/codesnippet/VisualBasic/how-to--programmatically-sort-data-in-worksheets_4.vb
        Dim fruitList As Excel.ListObject = CType(Application.ActiveSheet,  _
            Excel.Worksheet).ListObjects.AddEx(Excel.XlListObjectSourceType.xlSrcRange, _
            Application.Range("A1", "B2"))
        fruitList.Range.Sort( _
        Key1:=fruitList.ListColumns(1).Range, Order1:=Excel.XlSortOrder.xlAscending, _
        Key2:=fruitList.ListColumns(2).Range, Order2:=Excel.XlSortOrder.xlAscending, _
        Orientation:=Excel.XlSortOrientation.xlSortColumns, _
        Header:=Excel.XlYesNoGuess.xlYes)